src/Generators/ControllerGenerator.php

```diff
@@ -17,63 +18,30 @@
 use Blueprint\Models\Statements\SessionStatement;
 use Blueprint\Models\Statements\ValidateStatement;
 use Blueprint\Tree;
-use Illuminate\Filesystem\Filesystem;
 use Illuminate\Support\Str;
 
-class ControllerGenerator implements Generator
+class ControllerGenerator extends AbstractClassGenerator implements Generator
 {
-    const INDENT = '        ';
-
-    /**
-     * @var Filesystem
-     */
-    protected $filesystem;
-
-    private $imports = [];
-
-    /**
-     * @var Tree
-     */
-    private $tree;
-
-    public function __construct(Filesystem $filesystem)
-    {
-        $this->filesystem = $filesystem;
-    }
+    protected $types = ['controllers'];
 
     public function output(Tree $tree): array
     {
         $this->tree = $tree;
 
-        $output = [];
-
         $stub = $this->filesystem->stub('controller.class.stub');
 
         /** @var \Blueprint\Models\Controller $controller */
         foreach ($tree->controllers() as $controller) {
-            $this->addImport($controller, 'Illuminate\\Http\\Request');
-
+            $this->addImport($controller, \Illuminate\Http\Request::class);
             if ($controller->fullyQualifiedNamespace() !== 'App\\Http\\Controllers') {
-                $this->addImport($controller, 'App\\Http\\Controllers\\Controller');
+                $this->addImport($controller, \App\Http\Controllers\Controller::class);
```

Comment:
Referencing this via its static `class` property may yield an error if it doesn't exist. Hence the original string method.

Also, aside from this PR, it should probably respect the configured namespaces.